data/json/professions.json

```
@@ -1723,10 +1797,26 @@
     "skills": [ { "level": 2, "name": "rifle" } ],
     "items": {
       "both": {
-        "items": [ "army_top", "knit_scarf", "socks", "boots_steel", "pants_cargo", "jacket_flannel", "hat_hunting", "wristwatch" ],
+        "items": [
+          "army_top",
+          "knit_scarf",
+          "socks",
+          "boots_steel",
+          "pants_cargo",
+          "jacket_flannel",
+          "hat_hunting",
+          "wristwatch",
+          "single_sling"
+        ],
         "entries": [
           { "group": "charged_smart_phone" },
-          { "item": "compcrossbow", "ammo-item": "bolt_steel", "charges": 1, "contents-item": "shoulder_strap" },
+          {
+            "item": "compcrossbow",
+            "ammo-item": "bolt_metal",
```

```suggestion
            "ammo-item": "bolt_steel",
```
Was changed in #53629